tests/e2e/e2e_test.go

```diff
@@ -211,3 +212,33 @@ func (s *IntegrationTestSuite) TestStateSync() {
 	err = chain.RemoveNode(stateSynchingNode.Name)
 	s.NoError(err)
 }
+
+func (s *IntegrationTestSuite) TestExpeditedProposals() {
+	if !s.skipUpgrade {
+		s.T().Skip("this can be re-enabled post v12")
+	}
+
+	var wg sync.WaitGroup
+	chain := s.configurer.GetChainConfig(0)
+	node, err := chain.GetDefaultNode()
+	s.NoError(err)
+
+	node.SubmitTextProposal("expedited text proposal", sdk.NewCoin(appparams.BaseCoinUnit, sdk.NewInt(config.InitialMinExpeditedDeposit)), true)
+	chain.LatestProposalNumber += 1
+	node.DepositProposal(chain.LatestProposalNumber, true)
+	wg.Add(1)
+	totalTime := make(chan time.Duration, 1)
+	go node.QueryPropStatusTimed(&wg, chain.LatestProposalNumber, "PROPOSAL_STATUS_PASSED", totalTime)
```

Comment:
nit: to make it clearer
```suggestion
	go node.QueryPropStatusTimed(&wg, chain.LatestProposalNumber, "PROPOSAL_STATUS_PASSED", totalTimeChan)
```